Assets/Scripts/Bullet/BulletParticleManager.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BulletParticleManager : MonoBehaviour
+{
+    public GameObject Trail;
```

Comment:
Is there a reason these member variables are public? Should we make them private and then write getters for them? Or is having public members normal C# convention?

Reply:
It's public so that I can easily link this in the unity editor.